File: testprojects/master_python_mainone/pipelines/PYTHON_GCP/code/pythondoanything/graph/SetOperation_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from pythondoanything.config.ConfigStore import *
from pythondoanything.udfs.UDFs import *

def SetOperation_1(spark: SparkSession, in0: DataFrame, in1: DataFrame, ) -> DataFrame:
    return in0.unionAll(in1)

File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/everythingSG_1/SetOperation_1.scala
package graph.everythingSG_1

import io.prophecy.libs._
import udfs.UDFs._
import graph.everythingSG_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SetOperation_1 {

  def apply(context: Context, in0: DataFrame, in1: DataFrame): DataFrame =
    in0.unionAll(in1)

}
